AWS_FOR_FLUENT_BIT_VERSION

```diff
@@ -1 +1 @@
-2.23.1
\ No newline at end of file
+2.23.2
```

Comment:
Isn't this a major version because it includes an enhancement? The enhancement is minor, so maybe that's why you are making this a minor version bump. Minor bump makes sense to me, just something to think about.

Reply:
No I don't think it's a major version change. We don't see much change between last version and this version. So just a minor change. I would expect major version change like multiple line support feature and version 1.19 or at least several important bug fix or enhancement PRs.